core/arch/arm/mm/core_mmu_lpae.c

```diff
@@ -484,6 +484,45 @@ void core_init_mmu_prtn(struct mmu_partition *prtn, struct tee_mmap_region *mm)
 	}
 }
 
+/*
+ * In order to support 32-bit TAs we will have to find
+ * a TAs VA base in the region (0, 4GB).
+ * Due to OP-TEE design limitation, TAs page table should be an entry
+ * inside a level 1 page table.
+ *
+ * Available options are only these:
+ * - base level 0 entry 0 - [0GB, 512GB)
```

Comment:
replace closing parenthesis `)` with closing square brace`]`.

Reply:
`)` means excluding in this case.